pkg/storage/ingest/reader.go

```diff
@@ -341,6 +340,10 @@ func newReaderMetrics(partitionID int32, reg prometheus.Registerer) readerMetric
 			Help:    "The number of records received by the consumer in a single fetch operation.",
 			Buckets: prometheus.ExponentialBuckets(1, 2, 15),
 		}),
+		fetchesErrors: factory.NewCounter(prometheus.CounterOpts{
+			Name: "cortex_ingest_storage_reader_fetch_errors_total",
```

Comment:
Do we already have a metric with the total number of fetches, to compute a % of failing ones?

Reply:
maybe we can infer something from the franz-go metrics, but i prefer to avoid that (is a batch the same as a fetch?). I added a counter for the number of fetches too.